discord/guild.py

```diff
@@ -203,6 +203,10 @@ class Guild(Hashable):
         The guild's description.
     verification_level: :class:`VerificationLevel`
         The guild's verification level.
+    vanity_url_code: Optional[:class:`str`]
+        The guild's vanity url code, if any
+        
```

Comment:
Unfortunately, this is failing due to the trailing whitespace in this line. The suggestion I made didn't have the whitespace, though it was probably hard to tell. In the future, if you want to apply a suggestion from GItHub there's a dedicated button to do so:

![image](https://user-images.githubusercontent.com/1695103/156911579-ee15f2f4-b97e-46a7-a886-14dff9a31f59.png)

It's pretty handy! Usually projects don't like trailing whitespace, so setting up your editor to trim whitespace on save might be a desirable setting. You can find out how to do this by googling your editor.

Sorry for the annoyance on the formatting. Black touts itself as an "uncompromising formatter", so it tends to be very fussy.